src/BlockTypes/StoreNotices.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+use Automattic\WooCommerce\Blocks\Utils\StyleAttributesUtils;
+
+/**
+ * StoreNotices class.
+ */
+class StoreNotices extends AbstractBlock {
+
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'store-notices';
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array    $attributes Block attributes.
+	 * @param string   $content Block content.
+	 * @param WP_Block $block Block instance.
+	 *
+	 * @return string | void Rendered block output.
+	 */
+	protected function render( $attributes, $content, $block ) {
+		ob_start();
+		woocommerce_output_all_notices();
+		$notices = ob_get_clean();
+
+		if ( ! $notices ) {
+			return;
+		}
+
+		$classname = isset( $attributes['className'] ) ? $attributes['className'] : '';
+
+		return sprintf(
+			'<div class="woocommerce wc-block-store-notices %s">%s</div>',
```

Comment:
WordPress CS used to throw a warning/error about `Multiple placeholders should be ordered.` Not sure why it doesn't anymore. But I still think we should apply that practice. 

```suggestion
			'<div class="woocommerce wc-block-store-notices %1$s">%2$s</div>',
```